kentik_api_library/kentik_api/public/device_label.py

```diff
@@ -16,56 +16,46 @@ class DeviceItem:
 # pylint: disable=too-many-instance-attributes
 
 
+@dataclass
 class DeviceLabel:
-    # pylint: disable=too-many-arguments
-    def __init__(
-        self,
-        name: str,
-        color: Optional[str] = None,
-        id: Optional[ID] = None,
-        user_id: Optional[ID] = None,
-        company_id: Optional[ID] = None,
-        devices: Optional[List[DeviceItem]] = None,
-        created_date: Optional[str] = None,
-        updated_date: Optional[str] = None,
-    ) -> None:
-        # read-write
-        self.name = name
-        self.color = color
-
-        # read-only
-        self._id = id
-        self._user_id = user_id
-        self._company_id = company_id
-        self._devices = devices
-        self._created_date = created_date
-        self._updated_date = updated_date
-
-    # pylint: enable=too-many-arguments
+    # read-write
+    name: str
+    color: str
+
+    # read-only
+    _devices: List[DeviceItem] = field(default_factory=list)
+    _id: ID = DEFAULT_ID
+    _user_id: Optional[ID] = DEFAULT_ID
```

Comment:
I think that user_id shouldn't be optional.
The same in `labels_payload.GetResponse`

Does this field differ from e.g. `company_id` in that matter?

Reply:
This is surprise from API side:
`    {
        "id": 2595,
        "name": "GCP: gcp kentik integration",
        "color": "#599AD9",
        "user_id": null,
        "company_id": "74333",
        "devices": [],
        "created_date": "2020-10-08T07:43:13.488Z",
        "updated_date": "2020-10-08T07:43:13.488Z"
    },
`

Reply:
Ok, resolved